retrieval/target.go

```diff
@@ -88,11 +89,8 @@ func (t TargetHealth) value() model.SampleValue {
 }
 
 const (
-	// HealthUnknown is the state of a Target before it is first scraped.
```

Comment:
"Close closes the connection"

On Fri, Sep 11, 2015 at 4:42 PM, Fabian Reinartz <notifications@github.com>
wrote:

> In retrieval/target.go
> <https://github.com/prometheus/prometheus/pull/1064#discussion_r39314985>:
>
> > @@ -88,11 +89,8 @@ func (t TargetHealth) value() model.SampleValue {
> >  }
> >
> >  const (
> > -	// HealthUnknown is the state of a Target before it is first scraped.
>
> Probably – single line above const sounds good though. Just not that
> excited by doc strings consisting of spaced combinations of identifier
> names.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prometheus/prometheus/pull/1064/files#r39314985>.
>